src/fileActions.jsx

```diff
@@ -563,3 +520,137 @@ const setDirectoryName = (val, setName, setNameError, setErrorMessage) => {
         setNameError(null);
     }
 };
+
+export const fileActions = (path, files, selected, setSelected, clipboard, setClipboard, addAlert, Dialogs) => {
+    const currentPath = path.join("/") + "/";
+    const directory_name = path[path.length - 1];
+
+    const menuItems = [];
+
+    const createLink = (currentPath, files, selected) => {
+        Dialogs.show(
+            <CreateLinkModal
+              currentPath={currentPath} selected={selected}
+              files={files.sort((a, b) => (a.name.toLowerCase() > b.name.toLowerCase())
+                  ? 1
+                  : ((b.name.toLowerCase() > a.name.toLowerCase())
+                      ? -1
+                      : 0))}
```

Comment:
This is some unreadable stuff..

Reply:
It's okay-ish code-wise, but it doesn't make sense to me here. First of all, you can probably shorten this to `a.toLowerCase().localeCompare(b.toLowerCase())

But moreover, this seems like the wrong place. The "files in current directory" state of the main application should already be sorted (I assume). So why do you need to re-sort here? The dialog should show the same sorting as the main view.

This is just moved code, but cleaning this up would be a good follow-up.